js/lessonfilter.js

```diff
@@ -1,18 +1,22 @@
 function resetSort() {
+  /* Function to reset sort buttons */
+  $('#current-sort').text('date');
   $('#current-sort').removeClass().addClass("sort-desc");
   $("#sort-by-date").removeClass().addClass("sort desc my-asc");
   $("#sort-by-difficulty").removeClass().addClass("sort my-desc");
 }
 
 
 function applySortFromURI(uri, featureList) {
-
+  /* Function to update lesson-list using featureList and sort direction
+    - uses URI to generate sort directions
+  */
   console.log("applying sort from URI");
```

Comment:
Remove the console log functions now that we're ready to push this

Reply:
These were there originally (i.e. before I started working on search) but happy to remove them.

Reply:
Should I also remove where we console.log the new URI?